src/microbe_stage/MicrobeCamera.cs

```diff
@@ -164,22 +152,14 @@ public Vector3 CursorVisualWorldPos
 
     public override void _Ready()
     {
-        var material = GetNode<CsgMesh3D>("BackgroundPlane").Material;
-        if (material == null)
-        {
-            GD.PrintErr("MicrobeCamera didn't find material to update");
-            return;
-        }
-
-        materialToUpdate = (ShaderMaterial)material;
+        backgroundPlane = GetNode<BackgroundPlane>("BackgroundPlane");
```

Comment:
I think it would be better to make the backgroundPlane variable an `[Export]` and set it directly in the editor than like this.